source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib

```diff
@@ -36,7 +40,17 @@
                 <userDefinedRuntimeAttribute type="boolean" keyPath="positiveUseDefault" value="YES"/>
                 <userDefinedRuntimeAttribute type="boolean" keyPath="destructiveUseDefault" value="NO"/>
             </userDefinedRuntimeAttributes>
-            <point key="canvasLocation" x="-196" y="-88"/>
+            <point key="canvasLocation" x="-196" y="-88.15592203898052"/>
```

Comment:
Where did the incredibly precise decimals come from? :)

Reply:
hahaha.
my best guess is that my hand could have shaken and moved the button in the canvas. This is a single UI element, so theire relative position shouldn't matter.